src/optimise.jl

```diff
@@ -56,7 +74,18 @@ function minimise(f, X::T, tol=1e-3) where {T}
 end
 
 
-function maximise(f, X::T, tol=1e-3) where {T}
-    bound, minimizers = minimise(x -> -f(x), X, tol)
+function maximise(f, X::T; structure::Strategy=DefStructure ,tol=1e-3 ) where {T}
+    bound, minimizers = minimise(x -> -f(x), X, structure, tol)
     return -bound, minimizers
 end
+
+
+
+
+
+
```

Comment:
There is still excess white space (blank lines) here.

Reply:
ok done..!!